### PR TITLE
Week 9: To Do Application

### PR DESCRIPTION
**Pull request by apandyamob**
image

image

### CODE REVIEW

File: Week-9/to-do-app/src/container/ToDoList/index.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+const ToDoList = () => {
+  const [toDos, setToDos] = useState([]);
+
+  const handleNewToDoCreation = (text) => {
+    setToDos((toDos) => [...toDos, { text, id: toDos.length }]);
+  };
+
+  const handleToDoToggleCompletion = (id) => {
+    const newToDos = [...toDos];
+    newToDos.find((todo) => todo.id === id).isDone = !newToDos.find(
```
Comment (RajeshSivanesan) [inline Week-9/to-do-app/src/container/ToDoList/index.jsx]: @apandyamob Why this much complexity ? Can you add the data model you are using ?
Will todo have { id; text; isDone }

Can you simplify this ? Is it just isDone flag update to true ?

**Review by RajeshSivanesan — APPROVED** [review]

File: Week-9/to-do-app/src/container/ToDoList/index.jsx (hunk 1 above)
Comment (RajeshSivanesan) [reply Week-9/to-do-app/src/container/ToDoList/index.jsx]: @apandyamob one more thing, why two images for the tick mark. Is there any difference ?

Comment (apandyamob) [reply Week-9/to-do-app/src/container/ToDoList/index.jsx]: > @apandyamob one more thing, why two images for the tick mark. Is there any difference ?

both images are having different resolutions, images are used in the manifest file.

Comment (apandyamob) [reply Week-9/to-do-app/src/container/ToDoList/index.jsx]: > @apandyamob Why this much complexity ? Can you add the data model you are using ? Will todo have { id; text; isDone }
> 
> Can you simplify this ? Is it just isDone flag update to true ?

Upon clicking on todo text, I am toggling isDone value to true/false.. I have optimized the logic. Thanks @RajeshSivanesan

**Pull request merged**